services/platform/platform.go

```diff
@@ -88,7 +90,7 @@ func (s *Service) Connect(ctx context.Context, req *platformpb.ConnectRequest) (
 	}
 	pmmServerURL := fmt.Sprintf("https://%s/graph", settings.PMMPublicAddress)
 
-	nCtx, cancel := context.WithTimeout(ctx, platformAPITimeout)
+	nCtx, cancel := context.WithTimeout(ctx, s.platformAPITimeout)
 	defer cancel()
 
 	ssoParams, err := s.connect(nCtx, &connectPMMParams{
```

Comment:
is it possible to apply it to all outgoing requests but not just connect?

Reply:
Done.

Reply:
is this change still needed? I see the timeout is applied to http client in general

Reply:
Indeed not needed. Removed.